site/scripts/survey.js

```diff
@@ -0,0 +1,101 @@
+const surveyMain = document.querySelector("main");
+
+const formElement = document.createElement("form");
+formElement.action = "https://formspree.io/f/mrbkblny";
+formElement.method = "POST";
+
+const labels = ["What is your name", "What is your email", "Which whole number is the best?", "Which day is the best?"];
+const labelFor = ["name_field", "email_field", "best_whole_number", "best_day"];
+const inputTypes = ["text", "email", "number", "date"];
```

Comment:
These could be simplified into an array of JSON objects. Say something like:
```suggestion
const controlsOneToFour = [
{label: "What is your name", name: "name_field", fieldType: "text"},
{label:"What is your email", name: "email_field", fieldType: "email"},
{ label: "Which whole number is the best?", name: "best_whole_number",  fieldType: "number"}, 
{ label: "Which day is the best?", name: "best_day", fieldType: "date"},
];
```